app/DoctrineMigrations/Version20230807105004.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230807105004 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE tour ADD date DATE');
```

Comment:
From what I see when running `make migrations-diff`, an index should be added on this column. 

```php
$this->addSql('CREATE INDEX IDX_6AD1F969AA9E377A ON tour (date)');
```